components/CalendarNavigationBar/index.js

```diff
@@ -11,15 +11,31 @@ const StyledHeader = styled.header`
 const StyledLink = styled(Link)`
   color: inherit;
   text-decoration: none;
+  padding: 10px;
   ${(props) => props.withborder && `border-right: 1px solid black;`}
+  ${(props) =>
+    props.isactive &&
+    `
```

Comment:
If you want to set multiple css properties based on a prop (i.e. `isactive`) then i would consider the `css` helper function from styled-components

[See notes here](https://github.com/neuefische/hh-web-23-2/blob/main/sessions/react-styled-components/react-styled-components.md#adapting-based-on-props)

I would also consider using camel case for this prop. i.e. `isActive` instead of `isactive`.